assets/scripts/sentiment/SentimentSurvey.jsx

```diff
@@ -129,17 +129,19 @@ function SentimentSurvey ({ visible = false, onClose = () => {}, handleVote }) {
                 ))}
               </div>
               <p>
-                {score === null ? (
-                  <FormattedMessage
-                    id="sentiment.about-text"
-                    defaultMessage="This survey helps Streetmix learn how people feel about streets."
-                  />
-                ) : (
-                  <FormattedMessage
-                    id="sentiment.thank-you"
-                    defaultMessage="Thank you for participating in this survey!"
-                  />
-                )}
+                <a href="/survey">
```

Comment:
Let's not change the default text here because these strings are already out for translation. With the other comments addressed, I'm happy to merge this in without UI for now. I'll handle UI from here.